### PR TITLE
Add packager

### PR DESCRIPTION
**Pull request by nastasi-oq**
With this PR we automate and standardize the building procedure to generate oq-python3.5 package,
including also all the procedure to generate continuous integration local repositories.
Tests are green here: https://ci.openquake.org/job/zdevel_oq-python/6/

### CODE REVIEW

File: packager-guest.sh [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+done
+
+tar -xvf python3.5_3.5.4.orig.tar.gz -C "$PKG_DIR" --strip-components=1
+cp python3.5_3.5.4.orig.tar.gz oq-python3.5_3.5.4.orig.tar.gz
+
+cd "$PKG_DIR"
+
+# FIXME: remove exit
+# env
+# exit 0
```
Comment (daniviga) [inline packager-guest.sh]: Please remove

File: packager.sh [hunk 2]
```diff
@@ -0,0 +1,475 @@
...
+    GEM_EPHEM_USER="ubuntu"
+fi
+
+LXC_VER=$(lxc-ls --version | cut -d '.' -f 1)
+
+if [ "$LXC_VER" -lt 2 ]; then
+    echo "LXC >= 2.0.0 is required." >&2
+    echo "Hint: LXC 2.0 is available for Trusty from backports."
+    exit 1
+fi
```
Comment (daniviga) [inline packager.sh:59]: LXC 3.0 is out. we can start thinking about removing old code

**Review by daniviga — APPROVED** [review]
LGTM

File: packager.sh (hunk 2 above)
Comment (nastasi-oq) [reply packager.sh:59]: We will do it, and also think about LXD too.

**Pull request merged**